grass7/imagery/i.sentinel/i.sentinel.coverage/i.sentinel.coverage.py

```diff
@@ -214,17 +216,40 @@ def main():
     grass.message(_("Retrieving Sentinel footprints from ESA hub ..."))
     fps = 'tmp_fps_%s' % str(os.getpid())
     rm_vectors.append(fps)
+
+
     if not options['names']:
-        s_list = grass.parse_command(
-            'i.sentinel.download',
-            settings=settings,
-            map=area,
-            clouds=options['clouds'],
-            producttype=producttype,
-            start=options['start'],
-            end=options['end'],
-            flags='lb',
-            quiet=True)
+        i_sentinel_download_params = {
+            'settings': settings,
+            'map': area,
+            'clouds': options['clouds'],
+            'producttype': producttype,
+            'start': options['start'],
+            'end': options['end'],
+            'flags': 'lb',
+            'quiet': True}
+        i_sentinel_download_cmd = 'i.sentinel.download {}'.format(
+            ' '.join(["{!s}={!r}".format(k,v)
+            for (k,v) in i_sentinel_download_params.items()
+            if k not in ['flags', 'quiet']]))
+        if 'quiet' in i_sentinel_download_params:
+            i_sentinel_download_cmd += ' --q'
+        if 'flags' in i_sentinel_download_params:
+            i_sentinel_download_cmd += ' -{}'.format(
+                i_sentinel_download_params['flags'])
+        cmd = grass.Popen(
+            i_sentinel_download_cmd, shell=True,
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        resp = cmd.communicate()
+        if resp[0] != b'':
+            s_list = resp[0].decode('utf-8').strip().splitlines()
+        else:
+            error_msg = ""
+            for i in range(0, len(resp)):
+                error_msg += resp[i].decode('utf-8')
+            grass.fatal(
+                _("Error using i.sentinel.download: {}".format(error_msg)))
```

Comment:
I think the formatting should be
```suggestion
                _("Error using i.sentinel.download: {}").format(error_msg))
```
 (see also https://github.com/OSGeo/grass-addons/pull/416#discussion_r569927240)